setup.sh

```diff
@@ -15,6 +17,15 @@ sudo apt-get install -y postgresql postgresql-contrib
 echo "install requirements"
 pip3 install -r /vagrant/requirements.txt
 
-echo "running flask_init.py"
-export FLASK_APP=/vagrant/flask_init.py
-python3 -m flask run --host=0.0.0.0 >> /vagrant/log.log 2>&1 &
+echo "configuring database"
+sudo su postgres <<POSTGRESQL_COMMANDS
```

Comment:
IIR Postgeas has some command-line shortcuts for DB and user creation, could be cleaner to use those then inlined SQL.